<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfflineControl</class>
 <widget class="QDialog" name="OfflineControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="synchronousPushButton">
       <property name="text">
        <string>同步位置</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/refresh.ico</normaloff>:/images/refresh.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hidePushButton">
       <property name="text">
        <string>隐藏数值</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/convert.ico</normaloff>:/images/convert.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="initDriverPushButton">
       <property name="text">
        <string>初始化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="emergencyStopPushButton">
       <property name="text">
        <string>急停(ALT+A)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QComboBox" name="spdPosSelectComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>speed mode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>absolute position mode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>relative position mode</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="addRecordPushButton">
           <property name="text">
            <string>添加新命令</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/add.ico</normaloff>:/images/add.ico</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="nameLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>自定义命令命名</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QPushButton" name="execReverseSeqPushButton">
           <property name="text">
            <string>逆序执行所选</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="interValueSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="execSeqPushButton">
           <property name="text">
            <string>顺序执行所选</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>插值次数：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="cycleCheckBox">
           <property name="text">
            <string>是否循环</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="execSeqStopPushButton">
           <property name="text">
            <string>执行停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QPushButton" name="exportPushButton">
           <property name="text">
            <string>Export</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/save-as.png</normaloff>:/images/save-as.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="interPeriodSpinBox">
           <property name="suffix">
            <string>ms</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="fileNameLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="importPushButton">
           <property name="text">
            <string>Import</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/select.png</normaloff>:/images/select.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>间隔时间：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="runStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ControlTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControlTableView</class>
   <extends>QTableView</extends>
   <header>controltableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
